src/common/components/header/header.tsx

```diff
@@ -26,27 +29,122 @@ export const Header: React.FC<HeaderProps> = observer(() => {
     LoginStore.logout();
   };
 
+  const sidebarToggle = () => {
+    setSidebar(!sidebar);
+  };
+
+  useEffect(() => {
+    if (window.innerWidth < 1024) {
```

Comment:
I suggest extracting such numbers as 1024 into global constants so it can be used anywhere needed for the responsive design and also to make it more clear what it means.

Reply:
Yep. Thanks for the idea! :D